pkg/config/parse.go

```diff
@@ -25,6 +25,7 @@ type Config struct {
 	CloudRuntime         string `validate:"required" envconfig:"ATHENS_CLOUD_RUNTIME"`
 	FilterFile           string `envconfig:"ATHENS_FILTER_FILE"`
 	EnableCSRFProtection bool   `envconfig:"ATHENS_ENABLE_CSRF_PROTECTION"`
+	TraceExporterURL     string `envconfig:"TRACE_EXPORTER"`
```

Comment:
Should this also have the `ATHENS_` prefix for consistency? I remember that `envconfig` has a way to globally set a prefix, so it could be that the other two config options shouldn't be hardcoding the prefix.

Reply:
Yeah probably. I'll add it.